source/src/protocols/recon_design/MSDMover.cc

```diff
@@ -280,34 +281,50 @@ MSDMover::apply_mpi( core::pose::Pose & pose ) {
 	/// Find out my designable residues from my pose and my resfile
 	utility::vector1< core::Size > my_designable_residues = get_designable_residues( pose, this_nodes_resfile );
 
-	/// Make a string out of the AAs at my designable positions in the current state
-	std::string my_sequence = get_designable_sequence ( pose, my_designable_residues );
+	/// Make a string vector out of the AAs at my designable positions in the current state
+    utility::vector1< std::string > my_sequence = get_designable_sequence ( pose, my_designable_residues );
+
+    std::string pass_seq;
+    for( const std::string& resi_base_name: my_sequence) {
+        pass_seq += resi_base_name + " ";
+    }
 
 	/// Get the AAs at designable positions of the other states I need to cooperate with
-	utility::vector1<std::string> other_pose_sequences( n_procs );
+	utility::vector1< utility::vector1<std::string> > other_pose_sequences( n_procs );
 	for ( core::Size ii = 1; ii <= n_procs; ++ii ) {
 		if ( rank == ii ) {
 			for ( core::Size jj = 1; jj <= n_procs; ++jj ) {
-				if ( rank!=jj ) utility::send_string_to_node( jj-1, my_sequence ); // node ranks are 0-indexed
+				if ( rank!=jj ) utility::send_string_to_node( jj-1, pass_seq ); // node ranks are 0-indexed
 				else other_pose_sequences[jj] = my_sequence;
 			}
 		} else {
-			other_pose_sequences[ii] = utility::receive_string_from_node( ii-1 ); // node ranks are 0-indexed
+            std::string passed_seq;
+			passed_seq = utility::receive_string_from_node( ii-1 ); // node ranks are 0-indexed
```

Comment:
Combine this on a single line. (In general, when possible, try to declare variables on the same line you first initialize them.)